posthog/api/async_migration.py

```diff
@@ -9,11 +9,28 @@
     rollback_migration,
     trigger_migration,
 )
-from posthog.models.async_migration import AsyncMigration, MigrationStatus, get_all_running_async_migrations
+from posthog.models.async_migration import (
+    AsyncMigration,
+    AsyncMigrationError,
+    MigrationStatus,
+    get_all_running_async_migrations,
+)
 from posthog.permissions import IsStaffUser
 
 
+class AsyncMigrationErrorsSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = AsyncMigrationError
+        fields = ["id", "description", "created_at"]
+        read_only_fields = ["id", "description", "created_at"]
+
+
 class AsyncMigrationSerializer(serializers.ModelSerializer):
+    error_cnt = serializers.SerializerMethodField()
+
+    def get_error_cnt(self, async_migration: AsyncMigration):
+        return len(AsyncMigrationError.objects.filter(async_migration=async_migration))
```

Comment:
`<SELECT COUNT(*)>` instead of `len(<SELECT *>)`:
```suggestion
        return AsyncMigrationError.objects.filter(async_migration=async_migration).count()
```